frontend/tests/controllers-test/ChallengeInviteCtrl.test.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+describe('Unit tests for Challenge Invite Controller', function () {

```

Comment:
Use same styled casing as suggested in other unit tests. Let's keep it consistent.

Reply:
done